Assets/Script/Gameplay/GameManager.Loading.cs

```diff
@@ -211,6 +211,8 @@ private async void Start()
             // Loaded, enable updates
             enabled = true;
             IsSongStarted = true;
+            // Collect all garbage potentially generated  in library/venue setup, etc
+            System.GC.Collect();
```

Comment:
Might just be a nitpick on my part lol, but this particular placement of the GC call feels a little awkward/shoehorned. I'd prefer for this to be done in `LoadingContext.Wait`, right after all queued loading tasks are finished up. (Would also be good to put it in `GlobalVariables.LoadScene` in addition to this.)

Would also prefer not using the fully-qualified name here, add a `using System;` directive instead and change it to:

```suggestion
            // Collect all garbage potentially generated  in library/venue setup, etc
            GC.Collect();
```

Reply:
>but this otherwise doesn't relax the need to avoid allocation for frequently-called code.  

This is more relevant to "do not use linq when fixing up shaders when loading yarground because it can later be collected when song is already running" comments rather than something happening often.

and fwiw there are already things  that generate garbage literally every frame. see #979 for  an example